virt/virtual_machines/virt-accessing-vm-ssh.adoc

```diff
@@ -0,0 +1,84 @@
+:_content-type: ASSEMBLY
+[id="virt-accessing-vm-ssh"]
+= Using SSH to access virtual machines
+include::_attributes/common-attributes.adoc[]
+:context: virt-accessing-vm-ssh
+
+toc::[]
+
+You can configure an SSH connection to access virtual machines (VMs) by using the following methods:
+
+* `virtctl ssh` command
++
+You add a public SSH key to a VM before it boots for the first time. Then, you connect to the VM by using the `virtctl ssh` command.
++
+This method is useful for troubleshooting a VM because it requires little time to set up. The connection is slow, but it works consistently even if you are behind a firewall or if the cluster does not have a load balancer.
```

Comment:
The connection is not slow per se but it puts extra load on the control-plane of the cluster.

Reply:
I added a module about SSH connection considerations to capture the pros and cons of different methods.